object-search/README.md

```diff
@@ -17,9 +17,7 @@
 
 </p>
 
-In this example, we use `fasterrcnn_resnet50_fpn` for object detection  and then index them with `MobileNetV2`. You can use this demo system to index an image dataset and query the most similar object from it. 
-
-[![](.github/.README_images/7262e2aa.png)](https://get.jina.ai)
+In this example, we use `fasterrcnn_resnet50_fpn` for object detection and then index these cropped object images with `MobileNetV2`. You can use this demo system to index an image dataset and query the most similar object in those images. 
```

Comment:
I would emphasize that the expectrd input images for query are objects image and not general images since the indexed part is the cropped detrected image